### DISCUSSION TITLE
Street names always present and filters for objects

### DISCUSSION THREAD

**Discussion by stebett** [organicmaps/organicmaps#1316] [Ideas]
Hi! Since this is my first time here, I want to start congratulating all the developers, contributors and supporters of this project, and to thank you all for the effort you are making.

The application is _extremely_ aesthetically pleasant, fast and functional, and the only things that I wish were different are 

1. Street names always present on the screen (given that the street is large enough with that level of zoom)
2. Possibility of filtering out types of objects like restaurants and also street numbers

I feel like that would be a great improvement on readability.

I have no experience in app development, but if somebody wants to give a hint on what should be changed, I can try to do it.

**Comment by biodranik** [comment]
Street names usually are not drawn then something else is drawn in its place. If not, then it's a bug and should be fixed.

Hiding house numbers could be implemented as a pre-programmed, switchable "style" of some sort. What is your use case for this scenario?

**Comment by stebett** [comment]
![android2](https://user-images.githubusercontent.com/46384690/134036199-7576c314-f19c-47bb-9d8e-740be3b268d5.jpg)

As you can see from the screenshot, it would be useful to see the name of the street you have to turn in. 

Regarding the object and house numbers, I think having so many numbers on the screen can be confusing.

**Reply by biodranik** [reply]
It is a good usability idea: to always draw names of streets on the route. But we should carefully design how to draw it on top/near the dashed line.

**Reply by stebett** [reply]
I think that writing it as bus stops (like "Pyramides" in the screenshot) could work: black font but with a white outline, and the opposite for dark mode

**Reply by biodranik** [reply]
It's not about the color, but about positioning and readability.
